theme/templates/article.html

```diff
@@ -106,6 +106,7 @@ <h2 class="top-subheader">{{ article.headline }}</h2>
 
     {% if DISQUS_SITENAME and SITEURL and article.status != "draft" %}
     <section class="col-12 comments secondary">
+        <div class="addthis_inline_share_toolbox"></div>
```

Comment:
This does not belong here. Social sharing buttons need to be outside the HTML for comments. As written the social sharing buttons will be visible only if Disqus comments are enabled.